README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
> Here's a statement my understanding would predict to be true: there's a `Contravariant` instance for `Op`, but no `Functor`. Is that correct?

Yep!

> Might it not be more accurate to talk of the functor instance for functions of fixed domain?

Probably :wink:.

I'll let @scott-christopher respond to your other questions, as the lack of type variables is confusing me.